h2o-core/src/main/java/water/api/ModelMetricsHandler.java

```diff
@@ -402,10 +419,14 @@ public void compute2() {
             throw new H2OIllegalArgumentException("Model type " + parms._model._parms.algoName() + " doesn't support calculating Feature Contributions.");
           }
           Model.Contributions mc = (Model.Contributions) parms._model;
-          Model.Contributions.ContributionsOutputFormat outputFormat = null == s.predict_contributions_output_format ?
-                  Model.Contributions.ContributionsOutputFormat.Original : s.predict_contributions_output_format;
-          Model.Contributions.ContributionsOptions options = new Model.Contributions.ContributionsOptions().setOutputFormat(outputFormat);
-          mc.scoreContributions(parms._frame, Key.make(parms._predictions_name), j, options);
+          if (parms._top_n == 0 && parms._top_bottom_n == 0 && !parms._abs) {
```

Comment:
this would be more elegant if you used ContributionsOptions - then you wouldn't need 2 branches of execution

Reply:
Done